code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -19,6 +19,8 @@
 	var/portal_duration = 5 MINUTES
 	/// How long a target remains in the Syndicate jail.
 	var/prison_time = 4 MINUTES
+	/// How long an antagonist target remains in the Syndicate jail.
+	var/felony_time = 10 MINUTES
```

Comment:
I think it would be better to have this as a define, then just overriding prison_time before adding the timer
```suggestion
	#define ANTAG_CONTRACT_TIME 10 MINUTES
```